src/middlewared/middlewared/plugins/jail.py

```diff
@@ -111,59 +114,66 @@ def query(self, filters=None, options=None):
         return await self.middleware.call('jail.create_job', options)
 
     @private
-    @accepts(
-        Dict("options",
-             Str("release", required=True),
-             Str("template"),
-             Str("pkglist"),
-             Str("uuid"),
-             Bool("basejail", default=False),
-             Bool("empty", default=False),
-             Bool("short", default=False),
-             List("props", default=[])))
     @job()
     def create_job(self, job, options):
-
         verrors = ValidationErrors()
 
-        self.validate_ips(verrors, options)
+        try:
+            iocage = ioc.IOCage(skip_jails=True, jail=options['uuid'])
```

Comment:
Let's drop this and the next line (no internal calls from iocage, those can change interface at any time), and instead use the same process as used elsewhere in the plugin. This let's us chase iocage's internal change once instead of N times.

Do this:
```
    try:
        uuid, _, _ = self.check_jail_existence(jail, skip=False)

        if uuid:
            verrors.add(
                'uuid',
                f'A jail with uuid {options["uuid"]} already exists'
            )
            raise verrors
    except (SystemExit, RuntimeError):
        iocage = ioc.IOCage(skip_jails=True)
```
Then deindent the rest back. This lets us exit early above.